man/repart.d.xml

```diff
@@ -217,6 +217,16 @@
         automatically used.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>UUID=</varname></term>
```

Comment:
the commit msg should probably be changed too: `Uuid` →`UUID=`

Reply:
Oh, you are right of course, sorry!